shared/src/commands/validate/validate.ts

```diff
@@ -295,7 +279,7 @@ export async function validate(
 
         await loadMetaSchemas(ajv, metaSchemaPath);
 
-        logger.info(`Loading pattern from : ${jsonSchemaLocation}`);
+        logger.info(`Loading MARKER pattern from : ${jsonSchemaLocation}`);
```

Comment:
`MARKER`?

Reply:
Ah, yes - this was what I was adding/removing to determine if the multi-package `watch` was working with the custom `watch.mjs` script :) Will remove this prior to seeking final approval/merge.

Reply:
This is now removed - resolving.